src/KafkaW/Consumer.cpp

```diff
@@ -133,7 +133,7 @@ void Consumer::init() {
 
   RdKafka = rd_kafka_new(RD_KAFKA_CONSUMER, Configuration, ErrorStringBuffer,
                          ErrorStringSize);
-  if (!RdKafka) {
+  if (nullptr == RdKafka) {
```

Comment:
`RdKafka == nullptr` would be more conventional

Reply:
You don't like Yoda notation? :-)